app/src/lib/feature-flag.ts

```diff
@@ -21,6 +21,10 @@ function enableDevelopmentFeatures(): boolean {
   return false
 }
 
+export function enableProgressBarOnIcon(): boolean {
+  return enableDevelopmentFeatures()
```

Comment:
@billygriffin @outofambit I just noticed this feature @ChamodyaDias as a Windows user myself I'm super excited about it. However, I thought it was odd that it is `enableDevelopmentFeatures` instead of `enableBetaaFeatures` due to the conversations below.

Reply:
@say25 yep, that's a fair question! i'm planning to flip this feature flag before our next beta release (probably next week).

Reply:
Awesome would be excited/interested in testing and potentially implementing additional Progress Bars for different workflows

Reply:
@say25 i'm here for it!!